src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.dsde.rawls.webservice
+
+/**
+ * Created by tsharpe on 9/25/15.
+ */
+
+import org.broadinstitute.dsde.rawls.model.UserInfo
+import org.broadinstitute.dsde.rawls.openam.UserInfoDirectives
+import org.broadinstitute.dsde.rawls.workspace.WorkspaceService
+import spray.routing._
+
+trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDirectives {
+  lazy private implicit val executionContext = actorRefFactory.dispatcher
+  val workspaceServiceConstructor: UserInfo => WorkspaceService
+
+  val adminRoutes = requireUserInfo() { userInfo =>
+    path("admin" / "users") {
+      get {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.ListAdmins)
+      }
+    } ~
+    path("admin" / "users" / Segment) { (userId) =>
+      get {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.IsAdmin(userId))
+      }
+    } ~
+    path("admin" / "users" / Segment) { (userId) =>
+      put {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.AddAdmin(userId))
+      }
+    } ~
+    path("admin" / "users" / Segment) { (userId) =>
+      delete {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.DeleteAdmin(userId))
+      }
+    } ~
+    // This oddity requires some explanation:  This takes care of a chicken and egg problem
+    // for a brand new deployment of firecloud.  (There are no admin users, but you have to
+    // be an admin to add one.) This endpoint creates the admin google group, and puts one
+    // user into it, and you don't have to be an admin to call it.  Subsequent calls to this
+    // method after the very first one will fail, however, since the group will already exist.
+    // Thus the bootstrapping wormhole is "self-healing".
+    path("admin" / "users" / Segment / "seed") { (userId) =>
```

Comment:
i assumed that the deployment script would do a curl to set this up so that it's never forgotten and occurs promptly.  but i don't care.  i'll remove the endpoint before creating a PR, and we can work with devops to implement an alternative.